src/components/Activity/ActivityButton.js

```diff
@@ -0,0 +1,142 @@
+import React, { useCallback, useEffect, useRef, useState } from 'react'
+import { Spring, animated } from 'react-spring/renderprops'
+import {
+  ButtonIcon,
+  IconAlert,
+  Popover,
+  GU,
+  Tag,
+  springs,
+  useTheme,
+} from '@1hive/1hive-ui'
+import ActivityList from './ActivityList'
+import { useActivity } from '../../providers/ActivityProvider'
+
+const ActivityButton = React.memo(function ActivityButton() {
+  const theme = useTheme()
+  const [opened, setOpened] = useState(false)
+  const { activities, markActivitiesRead, unreadCount } = useActivity()
+  const containerRef = useRef()
+  const popoverFocusElement = useRef()
+
+  const handleToggle = useCallback(
+    () =>
+      setOpened(opened => {
+        if (opened) {
+          markActivitiesRead()
+        }
+        return !opened
+      }),
+    [markActivitiesRead]
+  )
+
+  const handleClose = useCallback(() => {
+    markActivitiesRead()
+    setOpened(false)
+  }, [markActivitiesRead])
+
+  useEffect(() => {
+    if (popoverFocusElement.current) {
+      popoverFocusElement.current.focus()
+    }
+  }, [activities])
+
+  return (
+    <React.Fragment>
+      <div
+        ref={containerRef}
+        css={`
+          width: ${7.25 * GU}px;
+          display: flex;
+          align-items: center;
+          height: 100%;
+          outline: 0;
+        `}
+      >
+        <ButtonIcon
+          element="div"
+          onClick={handleToggle}
+          css={`
+            height: 100%;
+            width: 100%;
+            border-radius: 0;
+            padding-left: ${0.5 * GU}px;
```

Comment:
When clicking the button, seems it´s not fully centered

```suggestion
            padding-left: ${0.75 * GU}px;
```